web_widget_bokeh_chart/static/src/js/web_widget_bokeh_chart.esm.js

```diff
@@ -0,0 +1,52 @@
+/* global document */
+import {markup, onMounted, onPatched, onWillStart, useRef} from "@odoo/owl";
+import {CharField} from "@web/views/fields/char/char_field";
```

Comment:
Add charField import and change export to:
```
export const bokehChartWidget = {
    ...charField,
    component: BokehChartWidget,
};
```
To fix the console error when debug mode is enabled